plugins/kubernetes-backend/api-report.md

```diff
@@ -228,6 +235,12 @@ export type KubernetesObjectTypes =
   | 'customresources'
   | 'statefulsets';
 
+// @alpha (undocumented)
+export interface KubernetesProxyServices {
```

Comment:
Wm can we not make a type for this? `getProxyServices` is oddly protected instead of private and I don't feel like there's any point in having this type being visible outward at all. I'd prefer for `getProxyServices` to just return literally `{ clustersSupplier: ... }`.

Well even better would be to make those methods private instead of protected of course. I'm not sure why things are this way since before, but seems worth starting to deviate from that pattern.

Reply:
Removed unnecessary interface